openfisca_us/variables/gov/states/in/tax/income/exemptions/in_base_exemptions.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_us.model_api import *
+
+
+class in_base_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN base exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states["in"].tax.income.exemptions
+        num_filers = tax_unit("num", period)
```

Comment:
I think we're looking to deprecate `num` since it's ambiguously named (sorry this isn't documented). I think this makes it easier though.
```suggestion
        return p.base.amount * tax_unit("tax_unit_size", period)
```
For posterity, I think we'd do this instead of `num`:
```python
filing_status = tax_unit("filing_status", period)
head_spouse_count = where(filing_status == filing_status.possible_values.JOINT, 2, 1)
```

Reply:
Gotcha. Will add the variable irs/tax_unit